examples/hello-world/hello-numpy-sag/requirements.txt

```diff
@@ -0,0 +1 @@
+nvflare~=2.4.0rc
```

Comment:
Actually this needs to be updated across everything, so for now maybe it would be better to keep it this so it is the same as everything else, so when we update, we can do it with a single find and replace